cmd/vfkit/main.go

```diff
@@ -121,6 +121,19 @@ func newVMConfiguration(opts *cmdline.Options) (*config.VirtualMachine, error) {
 		return nil, err
 	}
 
+	if opts.UseGUI && len(vmConfig.VirtioGPUDevices()) == 0 {
+		log.Warnf("--gui flag specified but no virtio-gpu device configured, automatically adding one")
+		dev, err := config.VirtioGPUNew()
+		if err != nil {
+			return nil, fmt.Errorf("failed to add virtio-gpu device: %w", err)
+		}
+		dev.(*config.VirtioGPU).UsesGUI = true
+		err = vmConfig.AddDevice(dev)
+		if err != nil {
+			return nil, fmt.Errorf("failed to add virtio-gpu device: %w", err)
+		}
```

Comment:
Should we also add virtio-input devices? It was odd to get to the login screen with gui, and not to have any way of typing text.